extractor/src/test/java/org/schabi/newpipe/extractor/utils/JavaScriptExtractorTest.java

```diff
@@ -0,0 +1,55 @@
+package org.schabi.newpipe.extractor.utils;
+
+import org.junit.jupiter.api.Test;
+import org.schabi.newpipe.extractor.exceptions.ParsingException;
+import org.schabi.newpipe.extractor.utils.jsextractor.JavaScriptExtractor;
+import org.schabi.newpipe.extractor.utils.jsextractor.Lexer;
+import org.schabi.newpipe.extractor.utils.jsextractor.Token;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.schabi.newpipe.FileUtils.resolveTestResource;
+
+public class JavaScriptExtractorTest
+{
+    @Test
+    public void testJsExtractor() throws ParsingException {
```

Comment:
Remove this unneeded `public` modifier. See [this Sonar rule](https://rules.sonarsource.com/java/RSPEC-5786) for more details.